homeassistant/components/shelly/entity.py

```diff
@@ -405,10 +383,6 @@ async def async_added_to_hass(self) -> None:
         """When entity is added to HASS."""
         self.async_on_remove(self.coordinator.async_add_listener(self._update_callback))
 
-    async def async_update(self) -> None:
-        """Update entity with latest info."""
-        await self.coordinator.async_request_refresh()
```

Comment:
Class was updated to use `CoordinatorEntity` base class which already includes `async_update` implementation